.github/workflow-template/main-override.yml

```diff
@@ -12,6 +12,8 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
+permissions: write-all
```

Comment:
Looks dangerous... If you want to upload artifacts, take a try on this?

```diff
 permissions:
   id-token: write
   contents: read # This is required for actions/checkout
+  packages: write
``` 

Ref. https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
also need create release permission?

anyway, the current release already contains a bug https://github.com/singularity-data/risingwave/issues/2249. I'll continue to work on this workflow after the holiday. Still hot-patching the release.

Reply:
That's great. I'll bump v0.1.7 next week and fix this together.